sqldelight-gradle-plugin/src/main/kotlin/app/cash/sqldelight/gradle/SqlDelightDatabase.kt

```diff
@@ -329,10 +329,5 @@ abstract class SqlDelightDatabase @Inject constructor(
     }
   }
 
-  private val Source.outputDir get() =
-    if (sources.size > 1) {
-      File(generatedSourcesDirectory, name)
-    } else {
-      generatedSourcesDirectory
-    }
+  private val Source.outputDir get() = File(generatedSourcesDirectory, name)
```

Comment:
I have no idea why we did this and I already changed this in another locally branch too.